ynput/ayon-ftrack
# AYON to ftrack sync: Base of service

## Changelog Description
Implemented base of new service `ayontof`.

## Additional review information
The name can change in future and the service might be merged with processor. The service does nothing, but already can listen to prepared event topics. The sync will be implemented in future PRs.

## Testing notes:
1. Run `ayontof` service.
2. Change something in AYON. Each AYON entity change should trigger `ftrack.sync` event.

Resolves https://github.com/ynput/ayon-ftrack/issues/159
